### ISSUE TITLE
Florisboard Theme

### ISSUE THREAD

**Issue by ghost** [ZeusOfTheCrows/gruvbox-everything#2]
Florisboard have completely revamped their Themer and it no longer supports json themes.

**Comment by ZeusOfTheCrows** [comment]
yeah i know, i'll write an updated theme at some point; but at the moment the number of elements and their identifiers is a bit volatile, i think
